Exception/FailureTransportNotListable.php

```diff
@@ -0,0 +1,13 @@
+<?php declare(strict_types=1);
+
+namespace KaroIO\MessengerMonitorBundle\Exception;
+
+use Symfony\Component\Messenger\Exception\RuntimeException;
+
+class FailureTransportNotListable extends RuntimeException
```

Comment:
Personal preference (and what's done in Symfony core) - I like the `Exception` suffix - `FailureTransportNotListableException`